front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/messages.ts

```diff
@@ -67,4 +67,30 @@ export default defineMessages({
     id: 'app.containers.Admin.reporting.components.ReportBuilder.Widgets.ChartWidgets.ActiveUsersWidget.ChartWidgetSettings.youNeedToSelectADateRange',
     defaultMessage: 'You need to select a date range first.',
   },
+  analyticsChartAriaLabel: {
+    id: 'app.containers.admin.ReportBuilder.charts.analyticsChartAriaLabel',
+    defaultMessage: 'Label',
+  },
+  analyticsChartDescription: {
+    id: 'app.containers.admin.ReportBuilder.charts.analyticsChartDescription',
+    defaultMessage: 'Description',
+  },
+  accessibility: {
+    id: 'app.containers.admin.ReportBuilder.charts.accessibility',
+    defaultMessage: 'Accessibility',
+  },
+  analyticsChartAriaLabelTooltip2: {
+    id: 'app.containers.admin.ReportBuilder.charts.analyticsChartAriaLabelTooltip2',
+    defaultMessage:
+      'Provide a label that describes the chart for improved accessibility',
```

Comment:
Minor: For any sentences, our convention is to end with a full stop, e.g. 

"Provide a label that describes the chart for improved accessibility."
"Provide a more detailed description of the chart and any relevant trends for improved accessibility."

You might still see sentences without periods in our app, but those are usually old strings before we aligned on this convention, or strings that weren't caught before release :)